### ISSUE TITLE
[Feature Request] use project level access tokens 

### ISSUE THREAD

**Issue by Brilator** [nfdi4plants/ARCitect#278]
The commit menu offers to add access tokens to circumvent the timeout issue. 
We might want to change the tokens to project-based (`https://git.nfdi4plants.org/<user>/<arc>/-/settings/access_tokens`), instead of user PATs (https://git.nfdi4plants.org/-/user_settings/personal_access_tokens). 

That seems securer and easier to track, change, delete on data hub, if not needed any more. 

However, that is currently not possible (at least I get a 403 when pasting a project token into ARCitect).

**Comment by JonasLukasczyk** [comment]
I just tested this and I was able to use a project token. Did you create one with all the necessary permissions? I will rename `Personal Access Token` to `Access Token`. Should I also open the project access token page instead of the personal access token page when someone clicks on the key icon?

**Comment by Brilator** [comment]
Right, I selected `API`, but did not realize I would also have to select a role, which is not the case for global PATs. 

Well, it's a bit tricky. Opening the project access token page will also only work, once the project exists. So the routine would be to push the ARC first without token and add another remote with token, if needed.

**Comment by JonasLukasczyk** [comment]
Then I would stick with the current version except the rename. Since project tokens are supported we should just mention this in the documentation.

**Comment by JonasLukasczyk** [comment]
fc39a60cc2f64d94376a2da7c3a52e8d13f00b37 renames `personal access tokens` to the more general `access token`
